### ISSUE TITLE
mobile platforms?

### ISSUE THREAD

**Issue by BorisMolch** [milkytracker/MilkyTracker#308]
sorry, what doesis mobile platforms mean?

**Comment by Deltafire** [comment]
Where is the phrase "mobile platforms" used?

**Comment by peterbart-lmi** [comment]
> Where is the phrase "mobile platforms" used?

I thought it's on the webpage, but there we use the term "portable platform".

**Comment by BorisMolch** [comment]
sorry, indeed "Supported on multiple platforms including portable devices" in the about section of the web page.

**Comment by peterbart-lmi** [comment]
This is a relic from the old days when MilkyTracker could run on a Windows CE (Windows Mobile) powered handheld device called PDA. It was compiled in a special mode that run the GUI in 320x240 resolution.

**Comment by SpecterSalmon** [comment]
It would be nice to use this on my Android tablet. There is one on the Play Store, but it denies all of my devices because they're too new of devices. Would just be a nice option to use it and make music when I'm at a card shop or at my in-law's place.

**Comment by Nine-H** [comment]
There are a number of linux based emulation handhelds that would be interesting to support. Stuff like the anbernic handhelds and the miyoo mini. Some of them are very capable aarch64 machines with multicore and tons of ram and make great grooveboxes. The problem is that milkytracker expects a PC keyboard and mouse. There is an LSDJ alike tracker called LittleGPTracker that has a great interface, but it uses a custom project format and not .xm module files.

**Comment by SpecterSalmon** [comment]
I agree, it'll be great to see a tracker that could run on more than just desktop PCs such as Linux or Windows. I mean, making music on my PS Vita and exporting it to use in Godot for a project later would be a hell of a workflow.
